proposals/0207-rpc-message-protection.md

```diff
@@ -221,4 +221,4 @@ An OEM can choose to require either all RPCs of the app or none RPCs of the app
 }
 ```
 
-Similar to the WWW is moving from HTTP to HTTPS, we believe the trend for the SDL is to move from un-encrypted messages to all messages encrypted in the future. However, not all existing head unit hardware are capable of encrypting all RPC messages with an acceptable performance. Updating the hardware may take a long time and with a high cost. At the same time, we do have a need to encrypt remote control related RPCs that have a high risk for the new apps on the existing hardware. The main reason we prefer the solution of encrypting select RPCs to the solution of encrypting all RPCs is we want it to work on the legacy hardwares.
+Similar to the WWW moving from HTTP to HTTPS, we believe the trend for SDL is to move from un-encrypted messages to all messages encrypted in the future. However, not all existing head unit hardware are capable of encrypting all RPC messages with an acceptable performance. Updating the hardware may take a long time and with a high cost and no guarantee of results. At the same time, we do have a need to encrypt remote control related RPCs that have a high risk for the new apps on the existing hardware. The main reason we prefer the solution of encrypting select RPCs to the solution of encrypting all RPCs is we want it to work on the legacy hardwares.
```

Comment:
Change to:
Similar to the WWW moving from HTTP to HTTPS, **the trend for SDL should be to** move from un-encrypted messages to all messages encrypted in the future. However, not all existing head unit hardware **is** capable of encrypting all RPC messages with an acceptable performance. Updating the hardware may take a long time and with a high **costs** and no guarantee of results. At the same time, **there is a need** to encrypt remote control related RPCs that have a high risk for the new apps on the existing hardware. 

Please update this sentence not using `we`: `The main reason we prefer the solution of encrypting select RPCs to the solution of encrypting all RPCs is we want it to work on the legacy hardwares.`

Reply:
updated